src/pasteup/grid.js

```diff
@@ -0,0 +1,75 @@
+// @flow
+import styled from 'preact-emotion';
+import {
+    tablet as tabletMq,
+    desktop as desktopMq,
+    leftCol as leftColMq,
+    wide as wideMq,
+} from './breakpoints';
+
+const columns = {
+    tablet: {
+        max: 12,
+        width: 40,
+        gutter: 20,
+    },
+    desktop: {
+        max: 12,
+        width: 60,
+        gutter: 20,
+    },
+    leftCol: {
+        max: 14,
+        width: 60,
+        gutter: 20,
+    },
+    wide: {
+        max: 16,
+        width: 60,
+        gutter: 20,
+    },
+};
+
+const breakpointMqs = {
+    tablet: tabletMq,
+    desktop: desktopMq,
+    leftCol: leftColMq,
+    wide: wideMq,
+};
+
+const gridStyles = (breakpoint, columnCount) => ({
+    [breakpointMqs[breakpoint]]: {
+        float: 'left',
+        width:
+            columnCount * columns[breakpoint].width +
+            (columnCount - 1) * columns[breakpoint].gutter,
+        paddingRight: columns[breakpoint].gutter,
+        ':last-of-type': {
```

Comment:
you can remove the need for this by setting the col padding to the left and adding a default -20 left margin on `Row` I think?